app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/ActionableAddressView.kt

```diff
@@ -172,49 +167,58 @@ fun ActionableAddressView(
             scope.launch { sheetState.show() }
         }
     }
-
     Box(modifier = modifier) {
-        Row(
-            modifier = Modifier
-                .combinedClickable(
-                    onClick = {
-                        actions?.let { popupActions ->
-                            when (val actionData = popupActions.primary.onAction()) {
-                                is OnAction.CallbackBasedAction -> actionData.onAction(context)
-                                is OnAction.ViewBasedAction -> viewBasedAction = actionData
-                            }
-                        }
-                    },
-                    onLongClick = { isDropdownMenuExpanded = true }
-                ),
-            verticalAlignment = Alignment.CenterVertically,
-            horizontalArrangement = Arrangement.spacedBy(RadixTheme.dimensions.paddingXSmall)
-        ) {
-            val inlineContentId = "icon"
-            val text = buildAnnotatedString {
-                append(actionableAddress.displayAddress)
-                append(" ")
-                appendInlineContent(inlineContentId)
-            }
-            val inlineContent = mapOf(
```

Comment:
Removing of inline content breaks display of full account address in account settings, and maybe in other places that I can't recall now. Intention of using this was that icon can appear as a part of text, not next to Text composable. See Figma for UI reference

Reply:
Oh man, thanks for reminding me this. I will fix this.